bases/node20.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "Node 20",
+
+  "compilerOptions": {
+    "lib": ["es2023"],
+    "module": "commonjs",
+    "target": "es2022",
```

Comment:
I think we probably want to bump target here also

Reply:
`ES2022` is the latest target

```shell
$ tsc --version
Version 5.0.4
$ tsc --target ES2023
error TS6046: Argument for '--target' option must be: 'es3', 'es5', 'es6', 'es2015', 'es2016', 'es2017', 'es2018', 'es2019', 'es2020', 'es2021', 'es2022', 'esnext'.
```

Reply:
Ace 👍🏻